### PR TITLE
[ENH] start implementing BIDS stats model transformers

### PR DESCRIPTION
**Pull request by Remi-Gau**
- [x] add support for `Add`, `Subtract`, `Divide`, `Multiply` operations (many more to go)

[List of transforms to implement](https://docs.google.com/document/d/1F_REhtYTWHBgASV-aoZh9GSV7ZhP0Deg4os4EP7MpM8/edit#bookmark=id.lfhhb42pu9bo)

### CODE REVIEW

**Comment by marcobarilari** [comment]
lol

File: src/bids_model/applyTransformersToEventsTsv.m [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  trialTypes = tsvContent.trial_type;
+  trialTypesList = unique(trialTypes);
+
+  for iTrans = 1:numel(transformers)
+
+    % TODO make transformers more general
+    % - assumes only trial types can be input
+    % - assumes a single input and output
+    % - assumes transformations are only on onsets
```
Comment (Remi-Gau) [inline src/bids_model/applyTransformersToEventsTsv.m:47]:
```suggestion
    % TODO make transformers more general
    % - assumes only trial types can be input (therefore means that transformations cannot be "chained")
    % - assumes transformations are only on onsets
```

**Comment by CerenB** [comment]
Now i'm afraid!

**Comment by Remi-Gau** [comment]
In brief: the BIDS stats model specification should in theory allow you to define a bunch of operation you want to perform on your "inputs" before you include them in the models.

The simplest example I have started implementing is: say you have an difference between the stimuli onsets written in your `events.tsv` and the "real" time when stimuli actually occurred (say the stimulus computer messed up the time stamps and you know they all came one second earlier than what was recorded).

You can either:
- rewrite ALL your `events.tsv` to change this (Oh joy!)
- add a few lines in your model JSON to remove that offset for all the conditions you are using.

Which one do you take?

**Comment by Remi-Gau** [comment]
Added a bit of doc about this in here (scroll down):

https://cpp-spm.readthedocs.io/en/dev/set_up.html#bids-model-json-files

**Pull request merged**